SConstruct

```diff
@@ -67,7 +67,6 @@ def buildServer(env):
                   'src/vcl/Image.cc',
                   'src/vcl/TDBImage.cc',
                   'src/vcl/Video.cc',
-                  'src/vcl/VideoData.cc',
```

Comment:
Did you not face the VideoData test problem here?

Reply:
Nope, because video testing is more on interface methods only.